ui/src/app/cohort-review/overview-page/overview-page.ts

```diff
@@ -39,6 +39,7 @@ export class OverviewPage implements OnInit, OnDestroy {
   isCancelTimerInitiated: any = false;
   domainTitle: '';
   trackClickedDomains = false;
+  buttonsDisableFlag = false;
```

Comment:
This seems to always be the same as the 'spinner' variable. If so, we can remove this and use 'spinner' instead

Reply:
Removed the spinner flag.